nextroute/model_options.go

```diff
@@ -1,8 +0,0 @@
-package nextroute
-
-import (
-	"github.com/nextmv-io/sdk/nextroute/schema"
-)
-
-// An Option configures a model.
-type Option func(Model, schema.Input) error
```

Comment:
Why did this get removed? We'll need them for Options that add code, such as custom value function.

Reply:
see comment in platform